src/renderer/components/query-result-table-virtualized.jsx

```diff
@@ -234,30 +258,51 @@ export default class QueryResultTable extends Component {
       );
     }
 
+    var self = this;
+
     return (
       <div>
         {previewModal}
 
         <ScrollSync>
           {({ clientHeight, clientWidth, onScroll, scrollHeight, scrollLeft, scrollTop, scrollWidth }) => {
-            
+
              return ( <div>
                 
                 {this.renderHeader()}
-                
-                <Grid
-                    columnWidth={this.getColumnWidth}
-                    columnCount={fields.length}
-                    height={30}
-                    cellRenderer={this.renderHeaderCell}
-                    className="grid-header-row"
-                    rowHeight={30}
-                    rowCount={1}
-                    width={tableWidth - scrollbarSize()}
-                    scrollLeft={scrollLeft}
-                >
 
-                </Grid>
+                {(() => {
```

Comment:
Is there a reason for self invoking function here? If it really has reason for that I guess including some comment would help understanding it in the future.